src/containers/Bounty/Bounty.module.scss

```diff
@@ -38,9 +38,10 @@
   padding: $m-spacing;
   background-color: $brand-purple-hover;
   border-radius: $base-border-radius;
-  text-align: center;
+  text-align: left;
   height: fit-content;
   white-space: nowrap;
+  border-radius: 5%;
```

Comment:
Let's remove this and keep the `border-radius` on line 40. In general, it's best to try and use the existing variables.

Reply:
@c-o-l-o-r @hshar7 right now I don't think that's something we can help all that much. Shorter titles are going to result in the payout box just taking up the majority of vertical space. In the new fora designs I've adjusted the spacing and we will also be adding additional lines to the title sections (like for indicating which fora community the bounty originates from) so that should help kill some of that extra space. If you want, we could make it so that the numerals and `ETH` elements are the same size (right now the numerals are bigger). That would also decrease the height slightly.